server/services/common_test.go

```diff
@@ -0,0 +1,31 @@
+// Copyright 2019 Google LLC
```

Comment:
`_test.go` is a suffix reserved for files that contain test functions. We might consider naming this something like `test_utils.go` or `test_common.go`.

Reply:
Done!